app/controllers/profiles_controller.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+# Routes requests for Profiles
+class ProfilesController < ApplicationController
```

Comment:
Is this a tautology

Reply:
Yes.

Reply:
just making sure. :)

Reply:
What comments do you typically put on controllers?

Reply:
If I have a non-resource-focused controller, I might write an explanation of why this isn't conforming to the usual resource-focused pattern. Otherwise, the controller is standard..